grizli/multifit.py

```diff
@@ -4003,7 +4008,7 @@ def drizzle_grisms_and_PAs(self, size=10, fcontam=0, flambda=False, scale=1, pix
                                       fcontam=0, fill_wht=True, ds9=None,
                                       mask_segmentation=mask_segmentation)
 
-            kern = h_kern[1].data[:, h['CRPIX1']-1-size:h['CRPIX1']-1+size]
+            kern = h_kern[1].data[:, int(h['CRPIX1'])-1-size:int(h['CRPIX1'])-1+size]
```

Comment:
I guess I was assuming that `CRPIX` was always integers, which isn't true (0.0 is the center of the pixel in the FITS standard, ±0.5 are the edges).  This is probably fine, but would be worth checking if it matters here (I think it could be `1024.5` for JWST?).  By that I mean that you're right that the slice needs to be integers, but check if the slice is offset by a pixel  one way or the other from expected.